### ISSUE TITLE
SQLResultsTool (Does not have enough permission to do it)

### ISSUE THREAD

**Issue by NightsDream** [dotCMS/core#17849]
Release DotCMS 5.2.2

### Describe the bug
Error when I using the sql result tools on container

![image](https://user-images.githubusercontent.com/20954805/72404442-5b9c2700-3790-11ea-96d5-c9609480f8e8.png)

### Steps to reproduce the behavior:
1. Create Templates, Container
2. Link it up the browser, templates, container
3. Paste this code on the container
```
<br /> Query: select * from identifier <br /><br /> 
  #set($query = "select * from identifier") 
  #set($resultsList = $sql.getSQLResults("default", "$!{query}", 0, 4)) Results: $resultsList
</p>
```

4. Go to dotcms5.2.2\dotserver\tomcat-8.5.32\webapps\ROOT\WEB-INF\classes\dotmarketing-config.properties and change the (ALLOW_VELOCITY_SQL_ACCESS_TO_DOTCMS_DB=false)(ENABLE_SCRIPTING=false)  to (ALLOW_VELOCITY_SQL_ACCESS_TO_DOTCMS_DB=true)(ENABLE_SCRIPTING=true)

5. Go to dotcms5.2.2\bin\system\src-conf\dotmarketing-config.properties and change the (ALLOW_VELOCITY_SQL_ACCESS_TO_DOTCMS_DB=false)(ENABLE_SCRIPTING=false)  to (ALLOW_VELOCITY_SQL_ACCESS_TO_DOTCMS_DB=true)(ENABLE_SCRIPTING=true)

6. Make sure is Scripting User (admin@dotcms.com)
![image](https://user-images.githubusercontent.com/20954805/72405063-7cfe1280-3792-11ea-8caf-5912e3893354.png)

![image](https://user-images.githubusercontent.com/20954805/72405114-aa4ac080-3792-11ea-98f4-4290aafbabf0.png)

UPDATE: I found out I have the same issue with this guy
#17469

**Comment by jcastro-dotcms** [comment]
R&D can create the PR based on Oscar's hotfix.

**Comment by wezell** [comment]
fixed in : #19746
